app/models/send_message.rb

```diff
@@ -0,0 +1,15 @@
+require 'rubygems' # not necessary with ruby 1.9 but included for completeness
+require 'twilio-ruby'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.